tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Tests/ITestRunner.cs

```diff
@@ -0,0 +1,6 @@
+namespace Azure.Sdk.Tools.Cli.Services.Tests;
+
+public interface ITestRunner
+{
+    public Task RunAllTests(string packagePath, TestMode mode, CancellationToken ct = default);
```

Comment:
We don't strictly need the test mode right now, but I anticipate we'll need it soon, so I put it in. If there objections I can take it out.